docs/api/cozy-client.md

```diff
@@ -2349,6 +2357,34 @@ Fetch and build an URL to open a note.
 | client | <code>object</code> | CozyClient instance |
 | file | <code>object</code> | io.cozy.file object |
 
+<a name="saveTimeSerie"></a>
+
+## saveTimeSerie
+Helper to save a time serie document.
+
+**Kind**: global constant  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| client | <code>object</code> | The CozyClient instance |
+|  | <code>Object</code> | The time serie attributes |
+
+<a name="fetchTimeSerieByIntervalAndSource"></a>
+
+## fetchTimeSerieByIntervalAndSource ⇒ <code>object</code>
+Helper to retrieve time series by their date interval and source.
+
+The starting date must be greater or equal while the ending date must
+be stricly less than the given startDate and endDate parameters.
+
+**Kind**: global constant  
+**Returns**: <code>object</code> - The time series found by the query  
```

Comment:
Are we sure it's not an array instead of an object ? Could we be a bit more precise on the type ? It will be useful for typechecking later.

Reply:
This is actually an object in JSON-API, eg
```
{
  data: [{timeserie1}, {timeserie2}]
}
```
Should we describe the data array?

Reply:
You could have a 

```
@typedef TimeSeries
@augments object
@property data {Array<GeoJSON>} 
```

or something like this ?

> Should we describe the data array?

We try now to have docstrings with types so that we can use typescript. So if we can use user definde types instead of only object, it will be helpful. See https://github.com/cozy/cozy-client/pull/878

Reply:
Is f1b3d28 ok?

Reply:
Should be better with fb3bfb5